pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -707,6 +719,16 @@ def parse_args_and_make_package(args=None):
                           'filename containing xml. The filename should be '
                           'located relative to the python-for-android '
                           'directory'))
+    ap.add_argument('--content-providers', dest='content_providers',
+                    help=('Add providers xml rules to the '
+                          'AndroidManifest.xml file. The argument is a '
+                          'filename containing xml. The filename should be '
+                          'located relative to the python-for-android '
+                          'directory'))
+    ap.add_argument('--add-xml-resource', dest='add_xml_resource',
```

Comment:
Didn't we add something else recently for moving stuff into the src/main/res folder?
I'm wondering if what we really want is a generic "copy this file/folder into src/main/res" argument, rather than many arguments for different subfolders.

Reply:
Sure, I can replace it with a more generic argument to copy files/directories into `src/main/res`.